test/e2e/specs/editor/various/list-view.spec.js

```diff
@@ -42,7 +42,7 @@ test.describe( 'List View', () => {
 			] );
 
 		// Drag the paragraph above the heading.
-		const paragraphBlockItem = await listView.getByRole( 'gridcell', {
+		const paragraphBlockItem = listView.getByRole( 'gridcell', {
```

Comment:
Had to remove this. ESLint would not allow me to commit.

Reply:
CC @kevin940726

Reply:
`geyByRole` doesn't return a promise so this is the right fix 👍 .